doc/sphinx/source/developer_guide/simulation_main.rst

```diff
@@ -0,0 +1,23 @@
+.. developer_guide simulation_main
```

Comment:
Don't you think the simulator pages should go to the `user_guide` part rather than `developer_guide` ?

Reply:
makes more sense indeed